### PR TITLE
GLT-2713: added Working with Boxes tutorial

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: 7-0-boxes.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+is-detailed: true
+
+---
+
+<div id="toc">
+Table of Contents
+<ol>
+    <li><a href="#logging_in">Logging In</a></li>
+    <li><a href="#box-data">Creating Some Test Data</a></li>
+    <li><a href="#box-new">Create a Box</a></li>
```
Comment (callunity) [inline 7-0-boxes.md]: Create -> Creating (for stylistic consistency)

File: _includes/box-tutorials.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+   Now two positions are selected, and the additional controls will change to reflect
+   this.
+1. Click one of the row labels - the letters on the left of the diagram. The entire
+   row will be selected.
+1. Hold the Ctrl/Command key on your keyboard and click another row label. Now both
+   rows are selected.
+1. Click one of the column labels - the numbers at the top of the diagram. The entire
+   column will be selected.
+1. Below the diagram, click the _Select All_ button. All positions will be selected.
+1. click the _Select Odd Columns_ button. Columns 1, 3, 5, 7, 9, and 11 will be selected.
```
Comment (callunity) [inline _includes/box-tutorials.md]: click -> Click

File: _includes/box-tutorials.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+1. In the _Search_ box, type the barcode of your first pool - `XX-A01` and click _Lookup_.
+1. The name and alias of the pool will appear in the _Results_ dropdown. Click _Update
+   Position_ to confirm and save the pool in the selected box position. The diagram is
+   updated to show the selected position filled.
+
+<a name="box-replace-single" href="#">top</a>
+
+# 6. Replacing items
+
+You can add an item to a position that already has an item. this will cause the existing
```
Comment (callunity) [inline _includes/box-tutorials.md]: `this` -> `This`

File: _includes/box-tutorials.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+1. Select any single position containing one of your pools.
+1. Click the _Remove Item_ button. A warning appears, telling you that the item's
+   position will be unknown. Click _Remove_ to confirm.
+
+<a name="box-remove-multiple" href="#">top</a>
+
+# 9. Removing multiple items
+
+1. Select the rest of the positions you've added pools to.
```
Comment (callunity) [inline _includes/box-tutorials.md]: Maybe `Select the rest of the filled positions.`?

File: _includes/box-tutorials.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+    * _Box Size_: 8 x 12 scannable
+1. Click _Save_ at the top right.
+
+<a name="box-selecting" href="#">top</a>
+
+# 4. Selecting Positions
+
+<img src="pics/box-selecting.png" id="figure">
+
+1. On the Edit Box page, scroll down to the Contents section. You will see a diagram
```
Comment (morgantaschuk) [inline _includes/box-tutorials.md]: Use italics to be consistent: _Edit Box_ page, _Contents_ section

**Pull request merged**